.idea/codeStyleSettings.xml

```diff
@@ -37,6 +37,34 @@
             </value>
           </option>
         </AndroidXmlCodeStyleSettings>
+        <MarkdownNavigatorCodeStyleSettings>
```

Comment:
❌  Please do not commit this (disable the markdown navigator plugin -- and the NDK too if you don't use it, the IDE will open up much faster ;) )